stablehlo/reference/ProcessGrid.h

```diff
@@ -51,10 +51,12 @@ struct ProcessId {
 };
 
 // StableHLO `process_group`.
-class ProcessGroup : public SmallVector<ProcessId> {};
+struct ProcessGroup : public SmallVector<ProcessId> {};
 
 // StableHLO `process_groups`.
-class ProcessGroups : public SmallVector<ProcessGroup> {};
+struct ProcessGroups : public SmallVector<ProcessGroup> {
+  ProcessGroups findAll(ProcessId processId);
```

Comment:
I think we should also rename this to `findProcess` to avoid ambiguity with `std::find`.